stock-plugins/aiverify.stock.robustness-toolbox/algorithms/robustness_toolbox/tests/e2e/test_e2e.py

```diff
@@ -0,0 +1,77 @@
+from pathlib import Path
+
+import pytest
+from aiverify_robustness_toolbox.algo_init import AlgoInit
+from test_engine_core.plugins.enums.model_type import ModelType
+
+binary_classification = {
+    "data_path": str(
+        "../../../user_defined_files/data/sample_bc_pipeline_credit_data.sav"
+    ),
+    "model_path": str("../../../user_defined_files/pipeline/bc_tabular_credit"),
+    "ground_truth_path": str(
+        "../../../user_defined_files/data/sample_bc_pipeline_credit_ytest_data.sav"
+    ),
+    "run_pipeline": True,
+    "model_type": ModelType.CLASSIFICATION,
+    "ground_truth": "default",
+}
+
+regression_non_pipeline = {
+    "data_path": str("../../../user_defined_files/data/sample_reg_donation_data.sav"),
+    "model_path": str(
+        "../../../user_defined_files/model/sample_reg_donation_sklearn_linear.LogisticRegression.sav"
+    ),
+    "ground_truth_path": str(
+        "../../../user_defined_files/data/sample_reg_donation_data.sav"
+    ),
+    "run_pipeline": False,
+    "model_type": ModelType.REGRESSION,
+    "ground_truth": "donation",
+}
+
+image_pipeline = {
```

Comment:
This is the result of the image pipeline e2e test:

```
{
    "results": {
        "num_of_perturbed_samples": 1,
        "org_performance": 1.0,
        "perturbed_performance": 1.0,
        "num_of_failed_perturbed_samples": 1,
        "perturbed_samples": [],
        "org_samples": [],
        "feature_names": [],
        "perturbed_pred": [
            9
        ],
        "original_pred": [
            [
                9
            ]
        ]
    }
}
```

It does not look very meaningful to me. Can you check if that's expected? The results for the tabular case seem fine.

Reply:
as discussed, based on output.schema.json only _["num_of_perturbed_samples", "org_performance", "perturbed_performance", "num_of_failed_perturbed_samples"]_ are the required fields.